Examples/CodePushDemoApp/demo.js

```diff
@@ -16,7 +16,7 @@ import CodePush from "react-native-code-push";
  * different check frequency, such as ON_APP_START, for a 'hands-off' approach where CodePush.sync() does not
  * need to be explicitly called. All options of CodePush.sync() are also available in this decorator.
  */
```

Comment:
Can you move this comment down to just above line 159, where it is used? And can you add the `checkFrequency` argument back in so that the behavior is equivalent to what it was before?